lib/FluxibleContext.js

```diff
@@ -173,46 +180,81 @@ FluxContext.prototype._initializeDispatcher = function initializeDispatcher() {
 /**
  * Returns the context for action controllers
  * @method getActionContext
+ * @param {Object} meta Metadata to keep track of the current flow of executing actions
  * @return {Object} Action context information
  */
-FluxContext.prototype.getActionContext = function getActionContext() {
-    if (this._actionContext) {
-        return this._actionContext;
-    }
+FluxContext.prototype.getActionContext = function getActionContext(meta) {
```

Comment:
After discussing offline, we decided while we can remove this parameter and move the objectAssign logic into `executeActionInternal` this is probably the best place for in since the `actionContext.executeAction` function's `this` depends on the `objectAssign` being done successfully.